task/views.py

```diff
@@ -62,5 +51,14 @@ def get(self, request, id=None):
         return Response(serializer.data)    
 
     
-
+class CheckTaskView(APIView):
+    serializer_class = TaskSerializer
```

Comment:
- This is not for Managers only. anyone should be able to check their tasks. 

- Status filter should be dynamic, not static ( "completed" )  .